src/qililab/result/quantum_machines_results/qm_result.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Qilimanjaro Quantum Tech
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""QuantumMachinesResult class."""
+import numpy as np
+
+from qililab.constants import QMRESULT, RUNCARD
+from qililab.result.counts import Counts
+from qililab.result.result import Result
+from qililab.typings.enums import ResultName
+from qililab.utils.factory import Factory
+
+
+@Factory.register
+class QuantumMachinesResult(Result):
+    """QuantumMachinesResult class. Contains the binning acquisition results obtained from Quantum Machines Manager execute() method.
+
+    Args:
+        raw_results (list): Raw results obtained from a Quantum Machines Manager.
+    """
+
+    name = ResultName.QM
+
+    def __init__(self, raw_results: list):
+        self.raw_results = raw_results
+
+    @property
+    def array(self) -> np.ndarray:
+        """Returns data as acquired from Quantum Machines Manager."""
+        return np.array(self.raw_results)
```

Comment:
This will definitely need changes. For each measurement, QM will return up to four `ndarray`: `I`, `Q`, `adc1` and `adc2` depending on the parameters given to the `measure` operation.

Reply:
Yes, those changes will be done once we have the rest of structure in place as improvements. Once the translation from/to config file is done and we can test more complex QUA programs.